### PR TITLE
use cleartax's eslint config

### LINKED ISSUE #44: Use @cleartax/eslint-config
We have the https://www.npmjs.com/package/@cleartax/eslint-config extensible config. Let's use this config

### PR DESCRIPTION
**Pull request by aditya81070**
Fixes #44

### CODE REVIEW

**Comment by maddhruv** [comment]
@aditya81070 can you resolve the `package.json` conflict?

**Comment by aditya81070** [comment]
@maddhruv I am on it.

File: package.json [hunk 1]
```diff
@@ -100,14 +100,14 @@
...
     "eslint-plugin-react": "^7.18.0",
     "eslint-plugin-react-hooks": "^2.3.0",
-    "husky": "^3.0.5",
+    "eslint-plugin-prettier": "^3.1.1",
+    "husky": "^4.0.10",
     "jest": "^24.9.0",
-    "lint-staged": "^9.2.5",
-    "prettier": "^1.19.1",
+    "lint-staged": "^10.0.1",
+    "prettier": "^1.18.2",
```
Comment (maddhruv) [inline package.json]: why `prettier` is downgraded?

**Comment by aditya81070** [comment]
@maddhruv  I have resolved the merge conflicts. Please review the PR again.

**Pull request merged**